polaris-react/src/components/SkeletonDisplayText/SkeletonDisplayText.tsx

```diff
@@ -12,14 +12,21 @@ export interface SkeletonDisplayTextProps {
    * @default 'medium'
    */
   size?: Size;
+  /**
+   * Adjusts size to be full width of container
+   * @default false
+   */
+  fullWidth?: boolean;
```

Comment:
We're trying to avoid boolean props where possible because they're not scalable

```suggestion
  maxWidth = `${number}ch` | `${number}%`;
```

Reply:
Sounds good, thanks @kyledurand, I just pushed out the latest, please let me know what you think especially in regards to the custom css var.